.github/ISSUE_TEMPLATE/Roadmap Feedback.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Copyright 2018-2023 The Khronos Group Inc.
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## Problem statement: 
+<!-- 
+Describe the problem you think should be addressed in future Vulkan roamaps
+-->
+
+## Use Case Example(s): 
+<!-- 
+List use case examples that illustrate the problem
+-->
+
+## (Optional) Suggested Solution(s) (via opening an MR on vulkan-docs repo and creating a Proposal Document) : 
+<!--
+Please open a PR on link:https://github.com/KhronosGroup/Vulkan-Docs[vulkan-docs] and create a link:https://github.com/KhronosGroup/Vulkan-Docs/blob/main/proposals/template.adoc[Proposal Document] describing suggested ways the Vulkan spec could change to address the problem (requires signed CLA)
```

Comment:
I think this could use a touch more explanation/rationale as to why a PR is required. Maybe something like:

_"For legal reasons we cannot accept technical design contributions, except as PRs (which are covered by our CLA). If you want to propose a technical solution, please open a PR on..."_